sdk/src/types/block/context_input/block_issuance_credit.rs

```diff
@@ -5,7 +5,7 @@ use derive_more::{Display, From};
 
 use crate::types::block::output::AccountId;
 
-/// A Block Issuance Credit Input provides the VM with context for the value of
+/// A Block Issuance Credit Input (BIC Input) provides the VM with context for the value of
```

Comment:
```suggestion
/// A Block Issuance Credit (BIC) Input provides the VM with context for the value of
```

Reply:
It didn't sound right to put the abbreviation in the middle 🤔

Reply:
Its common to but the abbreviation behind the words though, i approve of this suggestion :=)